docs/what.rst

```diff
@@ -0,0 +1,23 @@
+=====================
+What is a Certificate?
+=====================
+
+An SSL *certificate* contains a public key and a private key that work together to enable
```

Comment:
So maybe this should be re-worded as:
"A *certificate* is made up of a public key and a private key that work together to enable a web browser to communicate with your server securely over an encrypted SSL/TLS (transport layer security)
 connection."
I think mimicing the language in the HTTPS wikipedia article is probably a safe bet: https://en.wikipedia.org/wiki/HTTPS